src/theory/fp/theory_fp_rewriter.cpp

```diff
@@ -753,32 +778,43 @@ RewriteResponse maxTotal(TNode node, bool isPreRewrite)
     Assert(node.getKind() == kind::FLOATINGPOINT_TO_FP_SIGNED_BITVECTOR);
 
     TNode op = node.getOperator();
-    const FloatingPointToFPSignedBitVector &param = op.getConst<FloatingPointToFPSignedBitVector>();
+    const FloatingPointSize& size =
+        op.getConst<FloatingPointToFPSignedBitVector>().getSize();
 
     RoundingMode rm(node[0].getConst<RoundingMode>());
-    BitVector arg(node[1].getConst<BitVector>());
+    BitVector sbv(node[1].getConst<BitVector>());
 
-    FloatingPoint res(param.getSize(), rm, arg, true);
+    NodeManager* nm = NodeManager::currentNM();
 
-    Node lit = NodeManager::currentNM()->mkConst(res);
-    
-    return RewriteResponse(REWRITE_DONE, lit);
+    if (sbv.getSize() == 1)
```

Comment:
Can you give a quick comment here why this is a special case?

Reply:
It is not. It's just that @martin-cs assumed that it would not make sense to support it. A bit-vector `1` represents value `-1` of size `1`. Nothing special about it.

Reply:
I can add a comment that this is due to SymFPU not supporting it.